hippy/main.py

```diff
@@ -66,11 +56,26 @@ def setup_pypy_for_pyhyp(py_space):
   py_space.setattr(w_sys, py_space.wrap("executable"),
                    py_space.wrap(os.path.abspath(sys.argv[0])))
 
+def check_pypy_prefix_env(py_space):
+    # This stuff is needed to set sys.path and sys.exec_path
+    # CFFI needs this, for example.
+    from pypy.module.sys.initpath import pypy_find_stdlib
+    res = pypy_find_stdlib(py_space, None)
+
+    if res == py_space.w_None:
```

Comment:
this is nitpicking a bit, but one shouldn't compare Python W_Object's like this. Use: ``py_space.is_w(res, py_space.w_None)``